c2corg_ui/static/js/style/autoheight.js

```diff
@@ -0,0 +1,66 @@
+goog.provide('app.autoHeightDirective');
+goog.provide('app.AutoHeightController');
+
+goog.require('app');
+
+
+/**
+ * Apply automatic height depending on text to all textarea tags
```

Comment:
> It makes sense for the description attribute but not for the summary that should remain... short. 

If we want the summaries to be short, we should maybe add an explicit text "no loger than x characters" and a `max-length` attribute. The simple fact that a textarea will open according to the text is very handy imho. Have you tested it? if not, you should! :p

:+1:  ok for creating a special attribute to distinguish the textareas